RELEASE_NOTES.md

```diff
@@ -1,3 +1,6 @@
+### 11.0.0 - 2024-08-09
+* Fix unusable replay seed with FsCheck3
```

Comment:
Since this is a breaking change, the release notes should be more descriptive of the breaking change.
Perhaps something like 
```
Breaking Change: move FsCheckConfig.replay from int to uint64. 
Fixes issue where many FsCheck3 runs could not be replayed since the random seed is too large. 
Existing FsCheck 2 users should be able to use the same seeds values, but converted to uint64. 
Uint64 literals can be defined like `let iAm5 = 5UL` 
```

Reply:
Oh. The ReadMe also needs updated with the new replay type. Probably also with update guidance

Reply:
Updated thanks, let me know if there's anything I missed